src/client/apps/articles_list/components/articles_list_header.tsx

```diff
@@ -0,0 +1,73 @@
+import { Box, color, Flex, Sans, Tab, Tabs } from "@artsy/palette"
+import React, { Component } from "react"
+import { connect } from "react-redux"
+import styled from "styled-components"
+
+const Header = styled(Box)`
+  position: fixed;
+  top: 0;
+  left: 110px;
+  right: 0;
+  background-color: white;
+  z-index: 10;
+  border-bottom: 1px solid ${color("black10")};
+`
+
+const HeaderInner = styled(Flex)`
+  ${Flex} {
```

Comment:
These are nested Flex elements?

Reply:
Yeah, I'm trying to target the wrapper for the `Tab` component from Palette which is not exported.

Reply:
It may be worth adding that as a comment.